default.nix

```diff
@@ -59,10 +69,17 @@
     }
 
 , mkDerivation   ? null
+
+# **** 
+
 }:
 
+# *** ~let~ block (Custom abstractions)
+
 let
 
+# **** nix-store-src
```

Comment:
`hnix-store source` is better I would say.

Reply:
Yep. Maybe even rename the var in itself.

I would call for you for final review.